atf_eregs/static/regulations/css/less/typography.less

```diff
@@ -392,6 +392,10 @@ The `<ol>` tag needs to have the appropriate list type applied
 ```
 */
 
+.main-content {
+    overflow-x:scroll;
+}
```

Comment:
Paragraphs don't force an overflow - the text continues to wrap in the same manner.